src/stylesheets/dsdl/tokens.css

```
@@ -13,11 +13,12 @@
 /* Import webfonts */
 /* Space Grotesk: Variable weight, 400–700 */
 /* Source Code Pro: 400 only */
-@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400..700&family=Source+Code+Pro&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400..800;1,400..800&family=Source+Code+Pro&family=Space+Grotesk:wght@400..700&display=swap');
```

> I'd also recommend adjusting the default bold in body copy down from 700 to 600.

not sure whether she's suggesting that `600` is the boldest to use for Noto Sans or just a default bold...

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400..600;1,400..800&family=Source+Code+Pro&family=Space+Grotesk:wght@400..700&display=swap');
```

---

The DSDL does specify 800 (extra-bold) as an available option, however, that was in the context of Space Grotesk being used for everything. None of the core styles would have Noto Sans set in extra-bold. AND Christine mentioned wanting to use 600 as the standard bold in body copy, so limiting it to that might be a foolproof way to guarantee it 🤔

---

Oh, forgot to answer your second question. The two sets are one for regular and one for italics.